packaging/release/changelogs/changelog.py

```diff
@@ -211,6 +211,8 @@ def load_fragments(paths=None, exceptions=None):
     fragments = []
 
     for path in paths:
+        if path.startswith('.') or not path.endswith(('.yml', '.yaml')):
+            continue
```

Comment:
The sanity tests need to be updated to match this. They already handle the file extensions, but not the `.` prefix:

https://github.com/ansible/ansible/blob/4d675b6d876219780f234360b02ddee1ef3a0df1/test/sanity/code-smell/changelog.py#L18-L19

A similar change should be made to devel/stable-2.10 if the version of antsibull-changelog used there is updated to include the changes in https://github.com/ansible-community/antsibull-changelog/pull/33.